db/upgrade.php

```diff
@@ -33,8 +33,20 @@
  */
 function xmldb_poster_upgrade($oldversion) {
     global $CFG, $DB;
-
     $dbman = $DB->get_manager();
 
+    if ($oldversion < 2019091100) {
+        // Add new fields to poster table.
```

Comment:
The plural is confusing, we add a single field here. Can we have it like "Add a new field 'display' to the poster table."